Source/Core/DiscIO/NANDImporter.cpp

```diff
@@ -62,6 +61,9 @@ bool NANDImporter::ReadNANDBin(const std::string& path_to_bin)
 
   for (size_t i = 0; i < NAND_TOTAL_BLOCKS; i++)
   {
+    if (i % 1000 == 0)
```

Comment:
Especially when you could've easily used the old piece of code and send `i` plus `NAND_TOTAL_BLOCKS` to get a (more or less) accurate percentage for the NAND reading rather than just showing a (less meaningful) marquee (which is still better than no progress at all though!)

The rest of the code could've stayed as-is, with the current progress (which is now degraded to a marquee as well).